downstairs/src/main.rs

```diff
@@ -182,6 +182,45 @@ enum Args {
         bind_addr: SocketAddr,
     },
     Version,
+    /*
+    Measure an isolated downstairs
+    */
+    Dynamometer {
+        #[clap(long, default_value = "512", action)]
+        block_size: u64,
+
+        #[clap(short, long, name = "DIRECTORY", action)]
+        data: PathBuf,
+
+        #[clap(long, default_value = "100", action)]
+        extent_size: u64,
+
+        #[clap(long, default_value = "15", action)]
+        extent_count: u64,
+
+        #[clap(long, action)]
+        encrypted: bool,
+
+        // Number of writes to submit at one time to region_write
+        #[clap(short, long, action, default_value = "1")]
```

Comment:
Nit: you can use `default_value_t = 1` instead, which makes the default value typed at compile-time.

Reply:
nice, didn't know about that option! f1c3144